app/src/main/java/org/kinecosystem/kinit/server/api/OnboardingApi.kt

```diff
@@ -31,6 +31,8 @@ interface OnboardingApi {
                       @SerializedName("p2p_min_tasks") val p2p_min_tasks: Int,
                       @SerializedName("phone_verification_enabled") val phone_verification_enabled: Boolean,
                       @SerializedName("faq_url") val faq_url: String,
+                      @SerializedName("contact_us_url") val contact_us_url: String,
+                      @SerializedName("feedback_url") val feedback_url: String,
```

Comment:
I think we can remove these 3 URLs (faq_url, contact_us_url and feedback_url) from configuration and just use constants.

Reply:
not sure. I want to add this to the server, I don't think hardcoding those urls is a good idea, if any changes will force a new url we will have to update the client